mxcubeweb/core/adapter/beamline_adapter.py

```diff
@@ -37,13 +37,28 @@ def __init__(self, beamline_hwobj, app):
             gphl_workflow.connect(
```

Comment:
This connection and the function it calls should be removed. The only  place where it is used is obsolete code, that will be changed to sue gphlJsonParametersNeeded soon.